baseapp-profiles/baseapp_profiles/models.py

```diff
@@ -135,6 +133,14 @@ def get_graphql_object_type(cls):
 
         return ProfileObjectType
 
+    @property
```

Comment:
I don't think calculating this every time is a good idea since this field might be accessed through listing queries (list of comments, list of profiles, etc) where we need the profile name. This can lead to a [N+1](https://stackoverflow.com/questions/97197/what-is-the-n1-selects-problem-in-orm-object-relational-mapping) problem.  Having this as a calculated value also limits the possibility of filtering by name unless we annotate it to the queryset.

Can we please follow the PR description and use `name` as a cache field? The idea is that we update this field whenever an `organization` or `user` instance has changed.

Reply:
reverted to cached field name